static/app/views/explore/logs/useLogsQuery.tsx

```diff
@@ -56,13 +57,14 @@ export function useExploreLogsTableRow(props: {
   traceId: string;
   enabled?: boolean;
 }) {
+  const {isReady: pageFiltersReady} = usePageFilters();
   return useTraceItemDetails({
     traceItemId: String(props.logId),
     projectId: props.projectId,
     traceId: props.traceId,
     traceItemType: TraceItemDataset.LOGS,
     referrer: 'api.explore.log-item-details',
-    enabled: props.enabled,
```

Comment:
should this check props.enabled and pageFiltersReady

Reply:
Good catch man, yes, should check for both